WordPress/google-services.json-example

```diff
@@ -46,6 +46,48 @@
         }
       }
     },
+    {
+          "client_info": {
+            "mobilesdk_app_id": "1:123:android:abc",
+            "android_client_info": {
+              "package_name": "org.wordpress.android.jalapeno"
```

Comment:
```suggestion
              "package_name": "org.wordpress.android.prealpha"
```

The `package_name` value doesn't match the [`applicationId` value in the `build.gradle` file](https://github.com/wordpress-mobile/WordPress-Android/pull/11640/files?file-filters%5B%5D=.gradle&file-filters%5B%5D=.json-example&file-filters%5B%5D=.xml#diff-01b46d294a950c5acc9d967b2607983bR106).  We should update the `applicationId` value to `org.wordpress.android.jalapeno` in `build.gradle` ***or*** the `package_name` value to `org.wordpress.android.prealpha` in `google-services.json`.  If we change the `package_name` value, we need to make the change to the [other `package_name` instance below](https://github.com/wordpress-mobile/WordPress-Android/pull/11640/files?file-filters%5B%5D=.gradle&file-filters%5B%5D=.json-example&file-filters%5B%5D=.xml#diff-92d330fad535220493fa2f6dd96b0bd2R65).

Reply:
ohh, my bad. Good catch ;)!